backend/app/core/booking.py

```
@@ -691,76 +771,106 @@ def _process_stripe_refund(self, payment: Payment, amount: Decimal, reason: str
+        ).count()
+        cancelled_bookings = query.filter(
+            Booking.booking_status == BookingStatus.CANCELLED
+        ).count()
+
         # Get revenue statistics
-        revenue_query = query.filter(Booking.booking_status.in_([BookingStatus.CONFIRMED, BookingStatus.PAID]))
-        total_revenue = db.execute(
-            text("SELECT COALESCE(SUM(total_price), 0) FROM bookings WHERE booking_status IN ('confirmed', 'paid')")
-        ).scalar() or 0
-        
+        revenue_query = query.filter(
```

Ruff (0.11.9)

Local variable `revenue_query` is assigned to but never used

Remove assignment to unused variable `revenue_query`

(F841)